src/pdn/doc/define_pdn_grid.md

```diff
@@ -0,0 +1,104 @@
+## define_pdn_grid
+
+### Synopsis
+For specifying a power grid over the stdcell area
+```
+  % define_pdn_grid
+    -type stdcell \
+    [-name name] \
+    [-rails list_of_rail_specifications] \
+    [-straps list_of_strap_specifications] \
+    [-pins list_of_pin_layers] \
+    [-connect list_of_connected_layer_pairs] \
+    [-starts_with (POWER|GROUND)]    
```

Comment:
I think it would be easier to comprehend if it was broken into 3 commands, one for rails, one for straps, and one for rings.
And like we did for ICeWall, one call per definition, so for two different straps it would be two calls, etc.
Maybe something like:
`define_pdn_rail -name -width -pitch -layer -extend_to_rings` (pitch and width might be possible to get from the site and cells)
`define_pdn_straps -name -macro -width -pitch -offset -spacing -layers -pins -extend_to_rings -connect -starts_with`
`define_pdn_rings -name -width -offset -spacing -layers -pins -connect -starts_with`

Reply:
Ok - broken down the commands into smaller sub-units for defining the grid.